.github/workflows/tests.yml

```
@@ -80,6 +80,18 @@ jobs:
         unexpected = [ii for ii in text.split("\n") if "kitchen-sink" not in ii]
         assert len(unexpected) == 0
 
+    - name: Audit with Lighthouse
+      uses: treosh/lighthouse-ci-action@v8
+      with:
+        configPath: ".github/workflows/lighthouserc.json"
+        temporaryPublicStorage: true
+        uploadArtifacts: true
+        urls: |
+          reference/kitchen-sink/api.html
+          reference/kitchen-sink/paragraph-markup.html
+          reference/kitchen-sink/lists-and-tables.html
+          reference/special-theme-elements.html
+
   publish:
```

```suggestion
          reference/kitchen-sink/api.html
          reference/kitchen-sink/paragraph-markup.html
          reference/special-theme-elements.html

  publish:
```

so we can save some time on the number of runs